src/App/Fossa/ProjectInference.hs

```diff
@@ -103,13 +106,22 @@ inferSVN dir = context "Inferring project from SVN" $ do
     Just project -> pure project
   where
     toProps :: BL.ByteString -> [(Text, Text)]
-    toProps bs = mapMaybe toProp (Text.lines (decodeUtf8 bs))
+    toProps bs = mapMaybe toProp (linesWithCR (decodeUtf8 bs))
+
     toProp :: Text -> Maybe (Text, Text)
     toProp propLine =
       case Text.splitOn ": " propLine of
         [key, val] -> Just (key, val)
         _ -> Nothing
 
+-- Removes Windows `\r` from suffix if any
+-- We do this since: Text.lines does not remove \r.
+linesWithCR :: Text -> [Text]
```

Comment:
[Question] Should this be called `linesWithoutCR`? It is removing the carriage return.

Also, is there a particular reason not to use [replace](https://hackage.haskell.org/package/text-2.0/docs/Data-Text.html#v:replace):
```haskell
linesWithCR = lines . Text.replace "\r\n" "\n"
```
The only thing I can think of is that its time-complexity may be a bit worse, though it's a bit hard to reason about. I don't feel strongly that `replace` is better, besides maybe being shorter. What you have is fine too.